test/sharness/t0090-get.sh

```diff
@@ -20,7 +20,7 @@ test_ipfs_get_flag() {
   '
 
   test_expect_success "ipfs get $flag output looks good" '
-    printf "%s\n\n" "Saving archive to $HASH$ext" >expected &&
```

Comment:
This seems strictly better but is technically a breaking change. Any idea why this has changed?